InternTrack.Core.Api.Tests.Unit/Services/Foundations/Interns/InternServiceTests.Validations.RemoveById.cs

```diff
@@ -0,0 +1,109 @@
+﻿// ---------------------------------------------------------------
```

Comment:
Change the order of vno in your logic test as well. I could not leave comment there.

Reply:
What does VNO stand for?

Reply:
Neverminded figured it out.